CHANGELOG.md

```diff
@@ -8,13 +8,23 @@ Future Todo List
 - Make typography and utility classes silent extenders (so that they can be extended by components without importing all utility classes).
 - Deprecate modal and orderCard component styles in next major version as unused.
 
+v9.0.0-beta.0
+------------------------------
+*May 30, 2022*
+
+### Changed
+- **Breaking** Implement `@use` & `@forward` syntax in preference to `import` statements due to deprecation in `dart-sass`.
+  - README updated with new usage info. Documentation on fozzie.just-eat.com to be updated with full info on migration and usage.
```

Comment:
Is fozzie.just-eat.com still maintained? I know some of the documentation was moved across to vue.pie.design

Reply:
fixed.